packages/app/src/runner/aut-iframe.ts

```diff
@@ -35,6 +35,14 @@ export class AutIframe {
     return $iframe
   }
 
+  destroy () {
```

Comment:
This is new for single tab - we tear down the AUT between specs, right after we stop recording, but before we start the next spec.

For comparison of before/after, see: https://whimsical.com/spec-isolation-XiJyCyVUJmWQYRQbysgpP1